### ISSUE TITLE
isCancelled not being set in running Job

### ISSUE THREAD

**Issue by donpwilson** [yigit/android-priority-jobqueue#185]
Using the current version from jitpack IO.

When I looked at the code in debug mode, Android Studio reported that the first line of "query" in CancelHandler is "not executable code", although the next line is executable. The offending line is:

running = consumerManager.markJobsCancelled(tagConstraint, tags);

I hope this helps

**Comment by yigit** [comment]
can you clarify what the problem is?

`markJobsCancelled` has the following lines in it:

```java
if (constraint.matches(tags, holder.getTags())) {
                result.add(holder.getId());
                if (singleId) {
                    holder.markAsCancelledSingleId();
                } else {
                    holder.markAsCancelled();
                }
            }
```
So it will mark the job as cancelled.

**Comment by donpwilson** [comment]
After reflection, I realised the cause of this issue and the solution.

Despite appearances to the contrary in gradle and External Libraries, my app must have still been using a previous version (alpha2 or alpha3). Clean Project did not resolve the issue so I deleted all versions of alpha2 and alpha3 from my PC and then rebuilt the app.

It now runs markJobsAsCancelled correctly.

I'm happy that you close this issue
